src/web/components/Input/Input.scss

```diff
@@ -3,25 +3,24 @@
   justify-content: stretch;
   flex-direction: column;
   margin-top: 1rem;
-}
-
-.input-label {
-  margin-bottom: 0.5rem;
-  font-weight: 600;
-  font-size: 0.8125rem;
-  line-height: 2rem;
-}
 
-.input-container {
-  height: 45px;
-  box-sizing: border-box;
-  border: 1px solid var(--theme-input-border) !important;
-  border-radius: 5px;
-  padding: 0.5rem;
-}
+  .input-container {
```

Comment:
nit: it feels a bit unintuitive to me that an `input-container` is inside an `input-field`. But at the same time, I'm not sure what the best way to address that would be. e.g. I'm not sure if just flipping the names (i.e. a container has a field instead) would solve the problem. 

What do you think? Happy to leave as-is if no easy fix.

Reply:
Yes, I agree. But I find we have a few pages that are using input-container, flipping names needs to be careful. I will think if I can come up something with less file changes

Reply:
I change the `input-field` to `input-wrapper` I hope this make sense?

Reply:
makes sense to me!